packages/variable-editor/package.json

```diff
@@ -47,7 +47,7 @@
   "scripts": {
     "clean": "rimraf lib *.tsbuildinfo",
     "build": "tsc --build && tscp",
-    "package": "vite build",
+    "package": "npm run clean && vite build && tsc -p tsconfig.production.json && tscp",
```

Comment:
I think this is wrong.
The build already cleans at begin, then do a full tsc build. 
For the real build output however is the dts plugin in the vite.config reliable. 

If there are test files ending in the lib output then it is more likely that some references (e.g. the one in the root tsconfig or the one in the integration) do not refer the production tsconfig and so not filter this files out.